src/types/zetkin.ts

```diff
@@ -286,6 +286,7 @@ export interface ZetkinJourneyInstance {
   id: number;
   journey: {
     id: number;
+    singular_label: string;
```

Comment:
It might make sense to take this opportunity to also add the `plural_label: string` property that was also added as part of the same recent change to the API that introduced the `singular_label` field here.